toolkit/docs/formats/imageconfig.md

```diff
@@ -172,12 +172,12 @@ The image generated has users matching the values specified in Users.
 
 The table below are the keys for the users.
 
-|Key                |Type               |Restrictions
+Key                 |Type               |Restrictions
```

Comment:
Looks like an errant change.

Reply:
makes it so it isn't centered

Reply:
been bothering me for a while 😅

Reply:
In general, minimize # of deltas to only the changes that relate to the problem you are solving with this PR. Other things like style should be done in a separate PR.